release/release-it-release.json

```diff
@@ -1,7 +1,15 @@
 {
   "hooks": {
-    "before:init": ["git config user.email \"dont@write.here\"", "git config user.name \"Caring bot\""],
-    "after:bump": "git add ../../CHANGELOG.md; npx vsce package"
+    "before:init": [
+      "git config user.email \"dont@write.here\"",
+      "git config user.name \"Caring bot\""
+    ],
+    "after:bump": [
+      "git add ../../CHANGELOG.md",
+      "echo $THE_KEY > resources/TELEMETRY_KEY",
```

Comment:
Would the env variable $THE_KEY be visible in log here? I guess only secrets are obfuscated

Reply:
The value is not present in logs.
You can see an example here: https://github.com/grianbrcom/che-che4z-lsp-for-cobol/runs/1442316084?check_suite_focus=true in "Make a release" section.

Reply:
Nice, thanks